### DISCUSSION TITLE
Phantom wallet in desktop

### DISCUSSION THREAD

**Discussion by ryuk8000** [WalletConnect/walletconnect-monorepo#2471] [Developer Support]
When only phantom wallet is installed as an extension in desktop, then metamask is shown as installed wallet but when u click on metamask, phantom notification pops up. It is actually phantom but why installed wallet is shown as phantom.

**Comment by glitch-txs** [comment]
This behaviour was a Phantom decision on how to handle the EVM connection

https://phantom.app/blog/using-phantom-with-evm-apps

**Comment by finessevanes** [comment]
[Duplicate](https://github.com/orgs/WalletConnect/discussions/2470#discussion-5231816)
